src/Extensions/FileFormFactoryExtension.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace SilverStripe\SearchService\Extensions;
+
+use SilverStripe\Assets\Image;
+use SilverStripe\Control\RequestHandler;
+use SilverStripe\Core\Extension;
+use SilverStripe\Forms\CheckboxField;
+use SilverStripe\Forms\DatetimeField;
+use SilverStripe\Forms\Form;
+use SilverStripe\Forms\FormFactory;
+
+class FileFormFactoryExtension extends Extension
+{
+
+    public function updateForm(
+        Form $form,
+        ?RequestHandler $controller = null,
+        string $name = FormFactory::DEFAULT_NAME,
+        array $context = []
+    ): void {
+        $fields = $form->Fields()->fieldByName('Editor.Details');
+        $file = $context['Record'] ?? null;
+
+        if (!$fields || !$file || $file instanceof Image) {
```

Comment:
do we need an image type check? We can probably keep this generic as other kinds of files will want this functionality - eg pdfs